STT/whisper_stt_handler.py

```diff
@@ -137,4 +137,7 @@ def process(self, spoken_prompt):
         console.print(f"[yellow]USER: {pred_text}")
         logger.debug(f"Language Code Whisper: {language_code}")
 
+        if self.language is None:
+            language_code += "-auto"
+            
```

Comment:
There is no `self.language`. Did you test this ?

Reply:
My bad, forgot to commit a piece of code (`self.language = language`). I took the opportunity to make it a bit more clear and coherent with the Whisper MLX implementation. I also tested both local and server 👌